### PR TITLE
Update write_abstract_lef in README

### PR DESCRIPTION
**Pull request by niv280**
Signed-off-by: niv280 <niv280@gmail.com>

I update the write_abstract_lef description, let me know if it is clear enough or it needed to make some changes.

### CODE REVIEW

**Comment by rovinski** [comment]
@niv280 where is the "pitch" derived from? Is the pitch the same for every layer or is it different depending on the layer?

**Comment by niv280** [comment]
Different in every layer, I extract the pitch in that way:

```
tech_layer->getPitch();
```

File: src/README.md [hunk 1]
```diff
@@ -85,7 +85,24 @@ metal obstructions.
 
 ``` tcl
 read reg1.db
```
Comment (rovinski) [inline src/README.md:87]:
```suggestion
```

File: src/README.md [hunk 2]
```diff
@@ -85,7 +85,24 @@ metal obstructions.
...
-write_abstract_lef reg1_abstract.lef
+write_abstract_lef reg1_abstract.lef \
+		   [-bloat_factor shape_pitch_bloat_factor ] \
+		   [-bloat_occupied_layers ]
```
Comment (rovinski) [inline src/README.md:90]:
```suggestion
write_abstract_lef [-bloat_factor bloat_factor] \
                   [-bloat_occupied_layers] \
		   filename
```

File: src/README.md [hunk 3]
```diff
@@ -85,7 +85,24 @@ metal obstructions.
...
+write_abstract_lef reg1_abstract.lef \
+		   [-bloat_factor shape_pitch_bloat_factor ] \
+		   [-bloat_occupied_layers ] 	
+```
+### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-bloat_factor` | Specifies the bloat factor of the shapes in order for merging obstructions |
+| `-bloat_occupied_layers` | Specifies that the obstruction will include the entire layer |
```
Comment (rovinski) [inline src/README.md:97]:
```suggestion
| Switch Name | Description |
| ----- | ----- |
| `-bloat_factor` | Specifies the bloat factor used when bloating then merging shapes into LEF obstructions. The factor is measured in # of default metal pitches for the respective layer. A factor of `0` will result in detailed LEF obstructions |
| `-bloat_occupied_layers` | Generates cover obstructions (obstructions over the entire layer) for each layer where shapes are present |
```

File: src/README.md [hunk 4]
```diff
@@ -85,7 +85,24 @@ metal obstructions.
...
+
+### Examples
+```
+# Increase the shapes in 3 pitches (for every layer)
+write_abstract_lef reg1_abstract.lef -bloat factor 3
+
+# The obstructions will contain the entire layer of the nets
+write_abstract_lef reg1_abstract.lef -bloat_occupied_layers
```
Comment (rovinski) [inline src/README.md:105]:
```suggestion
read reg1.db

# Bloat metal shapes by 3 pitches (respectively for every layer) and then merge
write_abstract_lef -bloat factor 3 reg1_abstract.lef

# Produce cover obstructions for each layer with shapes present
write_abstract_lef -bloat_occupied_layers reg1_abstract.lef
```

**Review by rovinski — CHANGES_REQUESTED** [review]
I just added some more detail to the commands and separated out the generic interface from the example

**Comment by rovinski** [comment]
Format failure unrelated

File: src/README.md [hunk 5]
```diff
@@ -84,8 +84,26 @@ metal obstructions.
 
 
 ``` tcl
+write_abstract_lef [-bloat_factor bloat_factor] \
+                   [-bloat_occupied_layers] \
```
Comment (rovinski) [inline src/README.md:88]:
```suggestion
write_abstract_lef (-bloat_factor bloat_factor|-bloat_occupied_layers) \
```

**Review by rovinski — CHANGES_REQUESTED** [review]
Specify that options are mutually exclusive

**Review by rovinski — APPROVED** [review]

**Pull request merged**